.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+// Place your settings in this file to overwrite default and user settings.
+{
```

Comment:
I didn't notice you were using editorconfig. Do you want to have those settings in two places, or just use this for settings that aren't supported by editorconfig?